scripts/deployV1.1.0.ts

```diff
@@ -0,0 +1,173 @@
+import { Feeds, OBSSStorage, Profiles } from '../typechain'
+import { GSN_MUMBAI_FORWARDER_CONTRACT_ADDRESS } from '@big-whale-labs/constants'
+import { chains, ethAddressRegex } from './helpers/data'
+import { ethers, upgrades } from 'hardhat'
+import { utils } from 'ethers'
+import { version } from '../package.json'
+import deployContact from './helpers/deployContract'
+import prompt from 'prompt'
+
+async function main() {
+  const [deployer] = await ethers.getSigners()
+
+  const provider = ethers.provider
+
+  const { chainId } = await provider.getNetwork()
+  const chainName = chains[chainId]
+
+  // Deploy the contract
+  console.log('Deploying contracts with the account:', deployer.address)
+  console.log('Using network:', chainName)
+  console.log(
+    'Account balance:',
+    utils.formatEther(await deployer.getBalance())
+  )
+
+  const { isProduction } = await prompt.get({
+    properties: {
+      isProduction: { required: true, type: 'boolean', default: false },
+    },
+  })
+
+  const {
+    forwarder,
+    ketlTeamTokenId,
+    ketlAttestationAddress,
+    obssProxyAddress,
+    profilesProxyAddress,
+    feedsProxyAddress,
+  } = await prompt.get({
```

Comment:
Let's extract this `prompt.get` into `helpers`

Reply:
I think we'll have different `prompt.get` for each deployment, so it's not really reusable.